content/articles/how-to-ground-truth-label-your-datasets/index.md

```diff
@@ -0,0 +1,163 @@
+
+For this tutorial, we are going to load a 25 second video into the app. After doing this, we start by the ROI (region of interest) label by clicking on the label button.
+
+![labelling button](/engineering-education/how-to-ground-truth-label-your-datasets/ground3.png)
+
+When you click on this, the window below opens up.
+
+![new window](/engineering-education/how-to-ground-truth-label-your-datasets/ground4.png)
+
+Now, let us begin with that of `vehicle` then click `ok`. Note that you can label your data before defining the ROI. 
+
+Also, this app has a workspace shown below where all your label is stored, and you can see them.
+
+![workspace](/engineering-education/how-to-ground-truth-label-your-datasets/ground5.png)
+
+After completing this, label the vehicle with the bounding box. To label the data with the bounding box around them, click on the label at the top of the workspace.
+
+![vehicles labelled](/engineering-education/how-to-ground-truth-label-your-datasets/ground6.png)
+
+Let us label the sub-labels. Sub-labels are parts that are associated with a parent label. For example, we can have vehicle number plates or even wheels as the sub-labels. Let's, for example, name our vehicles `tarlight`. To label, the sub-labels, click on the `sub-labels` right next to the `label` button and then label them on your input using bounding boxes. 
+
+You can have as many bounding boxes as you want. If you click on the`sub-label` button, a new window like that of labeling prompts the user to input the name of the sub-labels.
+
+To make the labels on the input image, you have to select the bounding box of the parent label, and then you make your label inside the bounding box of this main bounding box. If you do not select the parent label first, you will not be able to make your sub-labels. 
+
+From the definition, sub-labels are parts of the parent data or ROI.
+
+![vehicles and the sublabels](/engineering-education/how-to-ground-truth-label-your-datasets/ground7.png)
+
+We can assign attributes for our labels and sub-labels. Attribute information makes up the metadata associated with the label. 
+
+In our example, we have the vehicle about the parent label. Now the attribute for vehicles can be vehicles type. 
+
+This attribute window allows the user to input a variety of types as a list. You can have as many as you want.
+
+![list of the vehicle types](/engineering-education/how-to-ground-truth-label-your-datasets/ground8.png)
+
+After making your attributes, you click `ok`, which appears on the left side of the window. For example, to label your vehicle types, you select the vehicle, move to the left, and choose the vehicle type below.
+
+![giving vehicle type](/engineering-education/how-to-ground-truth-label-your-datasets/ground9.png)
+
+We are happy with our label, but you notice that if we go to another frame, the labels are not there, as shown below. To go to the next frame, click on the forward circled in the image below:
```

Comment:
Can you be a bit more descriptive here. is "forward circled" a button or an icon?, kindly specify.